ext/readline/readline.c

```diff
@@ -305,7 +305,7 @@ PHP_FUNCTION(readline_info)
 			oldval = rl_erase_empty_line;
 			if (value) {
 				convert_to_long_ex(value);
-				rl_erase_empty_line = Z_LVAL_PP(value);
+				rl_erase_empty_line = Z_LVAL_P(*value);
```

Comment:
`rl_erase_empty_line` is not externed in [`editline/readline.h`](http://cvsweb.netbsd.org/bsdweb.cgi/src/lib/libedit/readline/readline.h?rev=1.37&content-type=text/x-cvsweb-markup&sortby=date), maybe we should add the necessary checks.